bcc/module.go

```diff
@@ -137,6 +139,12 @@ func (bpf *Module) Close() {
 		C.free(unsafe.Pointer(tpCategoryCS))
 		C.free(unsafe.Pointer(tpNameCS))
 	}
+	for _, vs := range bpf.perfEvents {
+		for _, v := range vs {
+			C.perf_reader_free(v)
+		}
+		// TODO: ?
```

Comment:
Not exactly sure what needs to happen here yet.

Reply:
From looking at the Python implementation (https://github.com/iovisor/bcc/blob/v0.6.0/src/python/bcc/__init__.py#L810-L844), `bpf_close_perf_event_fd` to close fd's from `bpf_attach_perf_event`. So `perfEvents  map[string][]unsafe.Pointer` can be changed to `perfEvents  map[string][]int`, I think.

Reply:
Updated, please have another look